src/Resources/config/sylius/ui.yaml

```diff
@@ -0,0 +1,10 @@
+sylius_ui:
+    events:
+        sylius.shop.register.form:
+            blocks:
+                monsieurbiz_anti_spam_shop_register:
+                    template: '@MonsieurBizSyliusAntiSpamPlugin/Shop/form_captcha.html.twig'
+        sylius.shop.contact.request.form:
+            blocks:
+                monsieurbiz_anti_spam_shop_contact:
+                    template: '@MonsieurBizSyliusAntiSpamPlugin/Shop/form_captcha.html.twig'
```

Comment:
Are you sure it's necessary? Don't we have a `render_rest` in the main form render?

Reply:
We have a render_rest but the template also manage error display and a message about the captcha policy

Reply:
Ok.